analyzers/src/SonarAnalyzer.CFG/CfgSerializer/CfgSerializer.cs

```diff
@@ -18,25 +18,32 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using SonarAnalyzer.CFG.LiveVariableAnalysis;
 using SonarAnalyzer.CFG.Roslyn;
 using SonarAnalyzer.CFG.Sonar;
 
-namespace SonarAnalyzer.CFG
+namespace SonarAnalyzer.CFG;
+
+public static partial class CfgSerializer
 {
-    public partial class CfgSerializer
+    public static string Serialize(IControlFlowGraph cfg, string title = "SonarCfg")
+    {
+        var writer = new DotWriter();
+        new SonarCfgWalker(writer).Visit(cfg, title);
+        return writer.ToString();
+    }
+
+    public static string Serialize(ControlFlowGraph cfg, string title = "RoslynCfg")
     {
-        public static string Serialize(IControlFlowGraph cfg, string title = "SonarCfg")
-        {
-            var writer = new DotWriter();
-            new SonarCfgWalker(writer).Visit(cfg, title);
-            return writer.ToString();
-        }
+        var writer = new DotWriter();
+        new RoslynCfgWalker(writer, new RoslynCfgIdProvider()).Visit(cfg, title);
+        return writer.ToString();
+    }
 
-        public static string Serialize(ControlFlowGraph cfg, string title = "RoslynCfg")
-        {
-            var writer = new DotWriter();
-            new RoslynCfgWalker(writer, new RoslynCfgIdProvider()).Visit(cfg, title);
-            return writer.ToString();
-        }
+    public static string Serialize(RoslynLiveVariableAnalysis lva, string title = "RoslynCfg")
```

Comment:
Not sure if it makes sense but I'd change the title to "RoslynCfgLVA" to avoid confusion in the future.
Like why I get this result here and another one in another case.
Like this, it might make it clear that this is a different CFG.